src/ngrest/base/NgRestModel.php

```diff
@@ -52,13 +52,13 @@
     public $i18n = [];
 
     protected $ngRestServiceArray = [];
-    
+
     /**
-     * @inheritdoc
+     * Attach behaviors at constructor to prevent override them with call of behaviors() at subclass model.
      */
-    public function behaviors()
+    public function __construct($config = [])
     {
-        return [
+        $this->attachBehaviors([
             'NgRestEventBehvaior' => [
                 'class' => NgRestEventBehavior::className(),
                 'plugins' => $this->getNgRestConfig()->getPlugins(),
```

Comment:
This line throws error

The scenario 'restcreate' does not exists in your scenarios list, have you forgote to defined the 'restcreate' in the scenarios() method?